### PR TITLE
api: Regenerate client interfaces for timestamp, ready, and simulate endpoints

### PR DESCRIPTION
**Pull request by algochoi**
This PR mostly uses the generator to generate the timestamp offset, ready, and simulate endpoints. 

Also implements tests for the timestamp and ready endpoint.

### CODE REVIEW

File: client/v2/algod/setBlockTimeStampOffset.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+// clock for the timestamp.
+type SetBlockTimeStampOffset struct {
+	c *Client
+
+	offset uint64
+}
+
+// Do performs the HTTP request
+func (s *SetBlockTimeStampOffset) Do(ctx context.Context, headers ...*common.Header) (response string, err error) {
+	err = s.c.post(ctx, &response, fmt.Sprintf("/v2/devmode/blocks/offset/%s", common.EscapeParams(s.offset)...), nil, headers, nil)
```
Comment (algochoi) [inline client/v2/algod/setBlockTimeStampOffset.go:21]: generator generated a GET here...

File: .test-env [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="master"
+SDK_TESTING_BRANCH="timestamp-tests"
```
Comment (algochoi) [inline .test-env:3]: TODO: revert back to master

File: .test-env [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Configs for testing repo download:
 SDK_TESTING_URL="https://github.com/algorand/algorand-sdk-testing"
-SDK_TESTING_BRANCH="master"
+SDK_TESTING_BRANCH="path-unit-tests"
```
Comment (algochoi) [inline .test-env:3]: todo: revert back to master

File: test/algodclientv2_test.go [hunk 4]
```diff
@@ -55,9 +55,12 @@ func AlgodClientV2Context(s *godog.Suite) {
 	s.Step(`^we make a GetStateProof call for round (\d+)$`, weMakeAGetStateProofCallForRound)
 	s.Step(`^we make a GetTransactionProof call for round (\d+) txid "([^"]*)" and hashtype "([^"]*)"$`, weMakeAGetTransactionProofCallForRoundTxidAndHashtype)
 	s.Step(`^we make a Lookup Block Hash call against round (\d+)$`, weMakeALookupBlockHashCallAgainstRound)
+	s.Step(`^we make a Ready call$`, weMakeAReadyCall)
 	s.Step(`^we make a SetSyncRound call against round (\d+)$`, weMakeASetSyncRoundCallAgainstRound)
 	s.Step(`^we make a GetSyncRound call$`, weMakeAGetSyncRoundCall)
 	s.Step(`^we make a UnsetSyncRound call$`, weMakeAUnsetSyncRoundCall)
```
Comment (winder) [inline test/algodclientv2_test.go:61]: There they are. Should be able to add in `unit.sync`.

File: client/v2/common/models/pending_transaction_response.go [hunk 5]
```diff
@@ -26,18 +26,18 @@ type PendingTransactionResponse struct {
...
 
-	// LocalStateDelta (ld) Local state key/value changes for the application being
-	// executed by this transaction.
+	// LocalStateDelta local state key/value changes for the application being executed
+	// by this transaction.
 	LocalStateDelta []AccountStateDelta `json:"local-state-delta,omitempty"`
 
-	// Logs (lg) Logs for the application being executed by this transaction.
+	// Logs logs for the application being executed by this transaction.
 	Logs [][]byte `json:"logs,omitempty"`
```
Comment (winder) [inline client/v2/common/models/pending_transaction_response.go:41]: Did the new node status fields fail to generate?

https://github.com/algorand/java-algorand-sdk/pull/555/files#diff-c8678c6feccbb68be3a4c002342c378832eaf4a03b2db6ff92747f13ef3d3874

Comment (algochoi) [reply client/v2/common/models/pending_transaction_response.go:41]: They were generated but I didn't add them here - I can do that in the next commit

Comment (algochoi) [reply client/v2/common/models/pending_transaction_response.go:41]: This is for the simulate endpoint

It looks like node status fields were already merged: https://github.com/algorand/go-algorand-sdk/blob/develop/client/v2/common/models/node_status_response.go

Comment (winder) [reply client/v2/common/models/pending_transaction_response.go:41]: oops! ty for the correction.

**Review by ahangsu — APPROVED** [review]
seems right on simulation part

**Review by winder — APPROVED** [review]

**Pull request merged**